docs/view_components/energy_system_component.rst

```diff
@@ -8,17 +8,16 @@ Attributes
 .. Please refer to the definition of what an attribute is in the tool_interface.rst fileg
 .. The properties should be filled in only if applicable.
 
-:ref:`energy-type-def`
-
-    Energy type linked to the different energy sectors (e.g. heat, electricity, ...)
+**Energy type**
+    ref:`energy-type-def` linked to the different energy sectors (e.g. heat, electricity, ...)
 
 
 **Component type**
-    Can be one of (Source, sink, transformer, storage, bus)
+    Can be one of [Source, Sink, Transformer, Storage, Bus, ...] .. we propably not going to only use oemof.solph basic components in the tool, so list needs to be extended
 
 
 **Unique id**
-    A positive integer number to identify the component uniquely within the list of energy system components of the :ref:`es_network-label`
+    A positive integer number to identify the component uniquely within the list of energy system components of the :ref:`es_network-label` .. why unique integer, rather than string label?
```

Comment:
we can have a label as well, the integer id allows easily to garantee the uniqueness (for every new component we bump the number by one. These attributes are characterising the view-component but are not necessarily rendered (ie visible to enduser)

Reply:
okay, i'm fine with that.